doc/classes/Environment.xml

```diff
@@ -9,7 +9,11 @@
 		- Glow
 		- Tonemap (Auto Exposure)
 		- Adjustments
-		These effects will only apply when the [Viewport]'s intended usage is "3D" or "3D Without Effects". This can be configured for the root Viewport with [member ProjectSettings.rendering/quality/intended_usage/framebuffer_allocation], or for specific Viewports via the [member Viewport.usage] property.
+		If the target [Viewport] is set to "2D Without Sampling", all post-processing effects will be unavailable. With "3D Without Effects", the following options will be unavailable:
+		- Ssao
+		- Ss reflections
```

Comment:
```suggestion
		- Ss Reflections
```

If these items are referring to Inspector sections, I suggest keeping the same capitalization.

Reply:
Bump. This should be the last update™ :)
Thanks a lot for going through the various iterations of review.

Reply:
I force pushed an update with this change.